hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/response/ContractCallResponse.java

```diff
@@ -0,0 +1,28 @@
+package com.hedera.mirror.test.e2e.acceptance.response;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import lombok.Data;
+
+@Data
+public class ContractCallResponse<T> {
+    private T result;
```

Comment:
Remove unused generic. The result is always a String. Leaving it could result in code smells.

```suggestion
public class ContractCallResponse {
    private String result;
```